delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/ScopeInitializationErrorStoreSuite.scala

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.projects
+
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.ScopeInitializationFailed
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{EntityType, ProjectRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.postgres.Doobie
+import ch.epfl.bluebrain.nexus.testkit.clock.MutableClock
+import ch.epfl.bluebrain.nexus.testkit.mu.NexusSuite
+import doobie.implicits._
+import doobie.postgres.implicits._
+import munit.AnyFixture
+
+import java.time.Instant
+
+class ScopeInitializationErrorStoreSuite
+    extends NexusSuite
+    with MutableClock.Fixture
+    with Doobie.Fixture
+    with Doobie.Assertions {
+
+  override def munitFixtures: Seq[AnyFixture[_]] = List(doobie, mutableClockFixture)
+  private lazy val xas                           = doobie()
+  private lazy val mutableClock: MutableClock    = mutableClockFixture()
+
+  private lazy val errorStore = ScopeInitializationErrorStore(xas, mutableClock)
+
+  private val project        = ProjectRef.unsafe("org", "proj")
+  private val entityType     = EntityType("test")
+  private val scopeInitError = ScopeInitializationFailed("boom")
+
+  private val scopeInitErrorRow = ScopeInitErrorRow(
+    1,
+    entityType.value,
+    project.organization.value,
+    project.project.value,
+    scopeInitError.reason,
+    Instant.EPOCH
+  )
+
+  test("Inserting an error should succeed") {
+    assertStoreIsEmpty >>
+      errorStore.save(entityType, project, scopeInitError) >>
+      errorStore.selectOne.assertEquals(scopeInitErrorRow)
+  }
+
+  case class ScopeInitErrorRow(
+      ordering: Int,
+      entityType: String,
+      org: String,
+      project: String,
+      message: String,
+      instant: Instant
+  )
+
+  implicit class ScopeInitErrorStoreOps(store: ScopeInitializationErrorStore) {
```

Comment:
I'm not sure what the implicit class is adding here, it's actually quite confusing that the store appears to have this method on it

Reply:
My thinking here was that I didn't want to add methods to the actual implementation that were only used in tests. However I agree it's not very successful, and it should be gone soon, if not in this PR then in the next one

Reply:
I see what you mean, but I would suggest that a normal method would be clearer than the implicit class

Reply:
As those information should be part of the new endpoint allowing to check that projects are correctly initialized.